build.gradle

```diff
@@ -526,7 +526,7 @@ project('standalone') {
         classpath += files('./pluginlib')
         doLast {
             def scriptFile = file getUnixScript()
-            scriptFile.text = scriptFile.text.replace('$APP_HOME/lib/pluginlib', '$APP_HOME/pluginlib/*')
+            scriptFile.text = scriptFile.text.replace('$APP_HOME/lib/pluginlib', '$APP_HOME/pluginlib/*:$APP_HOME/pluginlib')
```

Comment:
One clarification: why are changing project controller, project standalone, but not project segment store?

Reply:
The server side authplugin is loaded only in controller binary. Hence we need to update the controller binary. And as standalone run an inproc controller we need to make changes to standalone as well. Segmentstore is not involved directly in the auth flow. It just validates the authtoken.